src/txn/txn_recover.c

```diff
@@ -939,15 +926,15 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
 {
     WT_CONNECTION_IMPL *conn;
     WT_CURSOR *metac;
-    WT_DECL_ITEM(ckpt_lsn_str);
-    WT_DECL_ITEM(max_rec_lsn_str);
     WT_DECL_RET;
     WT_RECOVERY r;
     WT_RECOVERY_FILE *metafile;
     WT_TIMER checkpoint_timer, rts_timer, timer;
     wt_off_t hs_size;
+    char ckpt_lsn_str[WT_MAX_LSN_STRING];
     char *config;
     char conn_rts_cfg[16];
+    char max_rec_lsn_str[WT_MAX_LSN_STRING];
```

Comment:
Since they're related I'd suggest putting this one up on the line with `ckpt_lsn_str` separated by a comma.